examples/sample_pipeline/pipeline.py

```diff
@@ -63,3 +63,5 @@ def transform(self, dataframe: pd.DataFrame) -> pd.DataFrame:
     produces={"chunk_length": pa.int32()},
     arguments={"arg_x": "value_x"},
 )
+
+_.write(ref="write_to_file", arguments={"path": "./tmp"})
```

Comment:
`_` is used for unused variables. Since we're using it now, we should rename it :)

I also think we need to mount the directory?

Reply:
Didn't mount a directory cause we don't check/read the exported csv afterwards. The behaviour of the component is already tested within the component test. In the integration test we only test that we can add a write component to a pipeline.